templates/books-show.html

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-3 col-xs-12 thumbnail">
+      <img ng-src="{{booksShowCtrl.book.image}}" alt="book image">
+    </div>
+
+    <div class="col-md-9 col-xs-12">
+      <ul class="list-group form-inline">
+
+        <li class="list-group-item">
+          <a ng-href='books/{{booksShowCtrl.book._id}}'>
+            <h4 class='inline-header'>Book Title:</h4>
+            <span ng-hide="editing">{{booksShowCtrl.book.title}}</span>
+          </a>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.title" type="text" class="form-control input-md">
+
+        </li>
+
+        <li class="list-group-item">
+          <h4 class='inline-header'>Author:</h4>
+          <span ng-hide="editing">{{booksShowCtrl.book.author}}</span>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.author" type="text" class="form-control input-md">
+        </li>
+
+        <li class="list-group-item">
+          <h4 class='inline-header'>Release Date:</h4>
+          <span ng-hide="editing">{{booksShowCtrl.book.releaseDate}}</span>
+          <input ng-show="editing" ng-model="booksShowCtrl.book.releaseDate" type="text" class="form-control input-md">
+        </li>
+
+        <div class='panel-footer'>
+          <div class='panel-footer'>
+            <a href='/'>
+              <button class='btn btn-danger' ng-hide="editing" ng-click="booksShowCtrl.deleteBook(booksShowCtrl.book)">Delete Book</button>
+            </a>
+            <a href='/'>
```

Comment:
Do you really want to go back to the homepage whenever someone cancels or submits an edit?